app/tests/spec/views/cannot_create_account.js

```diff
@@ -52,5 +57,27 @@ function (chai, sinon, View, Relier) {
           assert.equal(view.$('.ftc').attr('target'), null);
         });
     });
+
+    it('has a working `Learn More` link with the default broker', function () {
+      return view.render()
+        .then(function () {
+          assert.isFalse(view.$('.show-visible-url').length > 0);
```

Comment:
Ooooh, I just found Chai's [lengthOf](http://chaijs.com/api/assert/#lengthOf-section) - this would be perfect!

```js
assert.lengthOf(view.$('.show-visible-url'), 0);
```